lighthouse-cli/index.js

```diff
@@ -139,22 +139,34 @@ if (cli.verbose) {
   flags.logLevel = 'error';
 }
 
-// kick off a lighthouse run
-lighthouse(url, flags, config)
-  .then(results => Printer.write(results, outputMode, outputPath))
-  .then(results => {
-    if (outputMode !== 'html') {
-      Printer.write(results, 'html', './last-run-results.html');
-    }
+function runLighthouse(addresses) {
+  // Process URLs once at a time
+  let address = addresses.shift();
+  if (!address) {
     return;
-  })
-  .catch(err => {
-    if (err.code === 'ECONNREFUSED') {
-      console.error('Unable to connect to Chrome. Please run Chrome w/ debugging port 9222 open:');
-      console.error('    npm explore -g lighthouse -- npm run chrome');
-    } else {
-      console.error('Runtime error encountered:', err);
-      console.error(err.stack);
-    }
-    process.exit(1);
-  });
+  }
+
+  lighthouse(address, flags, config)
+    .then(results => Printer.write(results, outputMode, outputPath))
+    .then(results => {
+      if (outputMode !== 'html') {
+        Printer.write(results, 'html', './last-run-results-' + Math.random() + '.html');
```

Comment:
Goooood question. I might be tempted to suggest `--output-path` is initially ignored if more than one URL is supplied. I'd probably want `--output-dir` more for this use-case (or, re-use `--output-path` supplying a directory) but wouldn't see folks needing an `--output-path-1`..`--output-path-n` type thing personally.